### ISSUE TITLE
There is no key to the two letter abreviations in the graph

### ISSUE THREAD

**Issue by mrjohnc** [mhausenblas/5stardata.info#35]
Unless I'm missing something there is no key to say what the two letter abreviations (below the stars) are on the graph at the top of the page.

![image](https://user-images.githubusercontent.com/7293392/33433923-d7dcf324-d5dd-11e7-9bf8-771ea4570580.png)

**Comment by jgkim** [comment]
@mrjohnc Thank you for raising the issue. I will look into how the image could be improved. In the mean time, you can find the legend to them below:

* OL: Open License
* RE: machine REadable
* OF: Open Format
* URI: Uniform Resource Identifiers
* LD: Linked Data

**Comment by mrjohnc** [comment]
Perfect, thanks :)

**Comment by akuckartz** [comment]
Also: not everybody knows what RDF and LOD stand for. I do, but I probably do not belong to the intended target audience.

**Comment by mrjohnc** [comment]
@akuckartz agreed, one option is linking to the Wikipedia articles for technical terms on the page
